test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -45,8 +45,8 @@
 	public static OptionalInt threadNumber = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessage = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessageSizeVar = OptionalInt.empty();
-	/** initial balance of payer account used for paying for performance test transactions */
-	public static OptionalLong initialBalance = OptionalLong.of(90_000_000_000_000L);
+	/** initial balance of account used as sender for performance test transactions */
+	public static OptionalLong initialBalance = OptionalLong.of(ONE_HBAR * 1_000_000L);
```

Comment:
This only increases the initial balances by 10,000 hbar. Will this be enough if we move to full 4-hr run?

Reply:
@ljianghedera On the 3 hour weekly tests, 90K hbars were sufficient for the test run. Now we are providing 1M hbars. It should be sufficient right?

Reply:
Actually it was 900K hbars for 3-hr run. I didn't expect it to run into the insufficient payer balance error for 3 hr run. So if we increase 100K hbars for another 1-hr, we may still run into this problem again. Since we are only in test, I suggest we increase the initial balances to 5M+ hbars to avoid this problem once and for all.

Reply:
Since the payer is not the sender. 1M should be sufficient.